### PR TITLE
Table: add context prop

### PR DESCRIPTION
**Pull request by QingWei-Li**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: examples/docs/zh-CN/table.md [hunk 1]
```diff
@@ -1008,7 +1115,7 @@
 | resizable | 对应列是否可以通过拖动改变宽度（需要在 el-table 上设置 border 属性为真） | boolean | — | true |
 | formatter | 用来格式化内容 | Function(row, column) | — | — |
 | show-overflow-tooltip | 当内容过长被隐藏时显示 tooltip | Boolean | — | false |
-| inline-template | 指定该属性后可以自定义 column 模板，参考多选的时间列，通过 row 获取行信息，JSX 里通过 _self 获取当前上下文。此时不需要配置 prop 属性。总共可以获取到 `{ row(当前行), column(当前列), $index(行数), _self(当前上下文), store(table store) }` 的信息。 | — | — |
+| inline-template | 指定该属性后可以自定义 column 模板，参考多选的时间列，通过 row 获取行信息。总共可以获取到 `{ row(当前行), column(当前列), $index(行数), store(table store) }` 的信息。 | — | — |
```
Comment (QingWei-Li) [inline examples/docs/zh-CN/table.md]: 等你写 🌚 反正英文文档也没弄 🌚

Comment (Leopoldthecoder) [inline examples/docs/zh-CN/table.md]: 你补上，我只写英文

File: examples/docs/zh-CN/table.md [hunk 2]
```diff
@@ -525,6 +538,12 @@
 
 <script>
   export default {
+    methods: {
+      handleClick() {
+        console.log(1);
```
Comment (Leopoldthecoder) [inline examples/docs/zh-CN/table.md]: 上面明明是 `console.log('click');`

**Comment by HerringtonDarkholme** [comment]
Vue now supports scoped template natively in 2.1. What's your opinion towards it? I think it can solve much edge cases such as context and variable scoping.

https://gist.github.com/yyx990803/faebe22e8763f5b17572b35ed96f52fe

**Comment by QingWei-Li** [comment]
@HerringtonDarkholme 
Yes, We will soon support the feature and as a recommended.

File: examples/docs/zh-CN/table.md (hunk 2 above)
Comment (erguotou520) [inline examples/docs/zh-CN/table.md]: 李大大面子可以啊

**Pull request merged**